Src/Plugins/PlaintextSecrets/PlaintextSecrets.json

```diff
@@ -4,15 +4,40 @@
       "Name": "DoNotExposePlaintextSecrets",
       "Id": "SEC1001",
       "Level": "Error",
+      "DefaultNameRegex": "(?i)\\.(azure|bat|cmd|config|cpp|cs|cscfg|definitions|dtsx|ini|java|jsx?|json|keys|loadtest|m|md|php|properties|ps1|psm1|pubxml|py|resx|sample|sql|ste|swift|test|tsx?|txt|waz|xml)$",
```

Comment:
>cpp [](start = 56, length = 3)

Add c, hxx, h, hpp? Let's continue to discuss the list of file extensions offline.